flexget/utils/search.py

```diff
@@ -11,7 +11,16 @@ def clean_symbols(text):
     result = text
     if isinstance(result, unicode):
         result = normalize('NFKD', result)
-    return re.sub('[ \(\)\-_\[\]\.]+', ' ', result).lower()
+    result = re.sub('[ \(\)\-_\[\]\.]+', ' ', result).lower()
+    
+    """Special cases"""
+    result = re.sub("(\'s)+", 's', result)
+    result = re.sub("(\'t)+", 't', result)
+
+    """Leftovers"""
+    result = re.sub(r"[^a-zA-Z0-9 ]", "", result)
```

Comment:
normalize('NFKD', text)) would remove á ã etc etc

result = re.sub('[ \(\)\-_\[\]\.]+', ' ', result).lower() will remove chars like - and _ and replace them with spaces

result = re.sub(r"[^a-zA-Z0-9 ]", "", result) will trim out the remaining special chars.

request #493 will add this functionality to kat, I'll make the same change to PiratBay